dist/lrm-mapzen.js

```diff
@@ -1400,6 +1402,14 @@ if (typeof module !== undefined) module.exports = polyline;
 			if (route.subRoutes) {
 				for(var i = 0; i < route.subRoutes.length; i++) {
 					if(!route.subRoutes[i].styles) route.subRoutes[i].styles = this.options.styles;
+					if (route.subRoutes[i].travel_type=='foot') {
+					  for (var s = 0; s < route.subRoutes[i].styles.length; s++)
+					    route.subRoutes[i].styles[s].dashArray = '4,10';
+					}
+					if (route.subRoutes[i].travel_type=='metro') {
```

Comment:
I also spent some times thinking about this circle marker on break points, I am not sure it should be default behaviour or not. we can make it option at least so users can turn off? (also why only `metro`?)